docs/getting-started-guides/vagrant.md

```diff
@@ -1,10 +1,9 @@
 ## Getting started with Vagrant
 
 ### Prerequisites
-1. Install latest version >= 1.6.2 of vagrant from http://www.vagrantup.com/downloads.html
+1. Install latest version >= 1.7.1 of vagrant from http://www.vagrantup.com/downloads.html
```

Comment:
Realisticaly, not sure. That's the version that I've verified works on both
of my dev boxes, so I upped them.

On Wed, Mar 4, 2015 at 3:15 PM, Eric Tune <notifications@github.com> wrote:

> In docs/getting-started-guides/vagrant.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5065#discussion_r25824788>
> :
>
> > @@ -1,10 +1,9 @@
> >  ## Getting started with Vagrant
> >
> >  ### Prerequisites
> > -1. Install latest version >= 1.6.2 of vagrant from http://www.vagrantup.com/downloads.html
> > +1. Install latest version >= 1.7.1 of vagrant from http://www.vagrantup.com/downloads.html
>
> out of curiosity, what breaks if you have vagrant older than 1.7.1?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5065/files#r25824788>
> .
>



-- 
*-- Fabio Yeon*